data/analytics-vendors-list.json

```diff
@@ -0,0 +1,94 @@
+{
+    "acquialift": "Acquia Lift",
+    "adobeanalytics": "Adobe Analytics",
+    "adobeanalytics_nativeConfig": "Adobe Analytics (native config)",
+    "afsanalytics": "AFS Analytics",
+    "alexametrics": "Alexa Internet",
+    "amplitude": "Amplitude",
+    "appsflyer": "AppsFlyer",
+    "atinternet": "AT Internet",
+    "baiduanalytics": "Baidu Analytics",
+    "blackcrowai": "Black Crow AI",
+    "blueconic": "BlueConic",
+    "bluetriangle": "Blue Triangle",
+    "browsi": "Browsi",
+    "burt": "Burt",
+    "byside": "BySide",
+    "captainmetrics": "Captain Metrics",
+    "chartbeat": "Chartbeat",
+    "clicky": "Clicky Web Analytics",
+    "colanalytics": "colanalytics",
+    "comscore": "comScore",
+    "cxense": "Cxense",
+    "deepbi": "Deep.BI",
+    "dynatrace": "Dynatrace",
+    "epica": "EPICA",
+    "euleriananalytics": "Eulerian Analytics",
+    "facebookpixel": "Facebook Pixel",
+    "gemius": "Gemius",
+    "gfksensic": "GfK Sensic",
+    "googleadwords": "Google Ads",
+    "googleanalytics": "Google Analytics",
+    "gtag": "gtag",
+    "N/A": "Google Tag Manager",
```

Comment:
Yes, It is referenced from [analytics-vendors-list.md](https://github.com/ampproject/amphtml/blob/main/extensions/amp-analytics/analytics-vendors-list.md#google-tag-manager).
We can also change it to an empty string, Is there anything else that you suggest?

Reply:
I think that an empty string would collide with the initial empty option i.e. "Select Type…" (they both would have a value of `''`).

Reply:
Agree, but I guess passing a `''` string will be more feasible, The [documentation](https://github.com/ampproject/amphtml/blob/main/extensions/amp-analytics/analytics-vendors-list.md#google-tag-manager) suggests keeping it empty. Also, will there be an issue if two options have the same value?

Reply:
@delawski  What we can do is let `Google Tag Manager` be as a default instead of "Select Type…" and hence we would have only one Option with `''` value.